src/server/server.ts

```diff
@@ -82,7 +82,7 @@ const startAppServer = (
       console.log(cyan, `myProxy is running on port ${port}!`)
       resolve(server)
     })
-  })
+  }).catch(error => console.error(`startAppServer: ${error}`))
```

Comment:
this is triggering the `new Promise` object so we should be catching here.

Reply:
instead, update `src/app.js` where this function is being called, and promise is being returned.